dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.ws.persistence.jakarta-1.0.feature

```diff
@@ -0,0 +1,21 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.ws.persistence.jakarta-1.0
```

Comment:
New features should not start with "com.ibm.ws"... nor should they include 'jakarta'.  'jakarta' was initially limited to the Jakarta API features, but those should now also be removed.

I believe the desired feature name here would have been "io.openliberty.persistence-2.0", but this would be confusing with the new name that will be used for the API, which is "io.openliberty.persistence.api-3.0" (i.e. io.openliberty.jakarta.persistence-3.0 needs to be renamed to this).

Perhaps this should get a more meaningful name like "io.openliberty.persistenceService-2.0"

Reply:
Yeah, and we might need to make a new pom file,t oo.  Perhaps just going with a new version is the better approach.

Reply:
Let me know what feature name you decide on -- or I'll just watch for a commit with the change -- so that I can update the corresponding persistent executor pull.  It was working well yesterday (that is, before the persistence service pull got backed out) and we were able to run two of our persistent executor buckets with Jakarta based on it.

Reply:
renamed it to `io.openliberty.persistenceService-2.0`